source/feature-guide/feature-importing.rst

```diff
@@ -413,6 +427,12 @@ attribute's default value instead.
 An empty ``order`` for the dataset will be handled as if no order was
 passed in.
 
+An empty ``hidden`` for the dataset, will assume a flat order for all the
```

Comment:
If "hidden" is omitted, the old API still works, right? That is, mark hidden variables as discarded: true, and order them if you want by inclusion in "order".

Reply:
Yes, the existing tests continue working normally.

Reply:
So this is wrong. An "empty `hidden`" member means there are no hidden variables. Alternatively, if "hidden" is omitted, hiddenness is determined as always before by the "discarded" attributes, and if those variables are included in "order", the folder structure for the hidden variables will be constructed based on their position in "order". If not included in "order", they'll be flat.